packages/spectral/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prismatic-io/spectral",
-  "version": "10.1.1",
+  "version": "10.2.0",
```

Comment:
No corresponding version bump in `package-lock.json`?

Reply:
I re-ran `yarn` and it didn't modify any lockfiles.